client/js/render.js

```diff
@@ -7,7 +7,7 @@ const renderPreview = require("./renderPreview");
 const utils = require("./utils");
 const sorting = require("./sorting");
 const constants = require("./constants");
-const condense = require("./condensed");
+const condensed = require("./condensed");
```

Comment:
I think this was coming from when I couldn't decide whether to say "condense" or "condensed".